samza-core/src/main/scala/org/apache/samza/config/MetricsConfig.scala

```diff
@@ -49,7 +49,7 @@ class MetricsConfig(config: Config) extends ScalaMapConfig(config) {
 
   def getMetricsSnapshotReporterStream(name: String): Option[String] = getOption(MetricsConfig.METRICS_SNAPSHOT_REPORTER_STREAM format name)
 
-  def getMetricsSnapshotReporterInterval(name: String): Option[String] = getOption(MetricsConfig.METRICS_SNAPSHOT_REPORTER_INTERVAL format name)
+  def getMetricsSnapshotReporterInterval(name: String): String = getOption(MetricsConfig.METRICS_SNAPSHOT_REPORTER_INTERVAL format name).getOrElse("60")
```

Comment:
Should update config docs with default value.

Reply:
Maybe return int here directly instead of toInt later. Not sure why this was a string.